src/lib/packet.ts

```diff
@@ -1,5 +1,4 @@
-export class ServerPacket {
-
+export class Packet {
     private readonly type: string;
     private readonly data: unknown;
```

Comment:
I don't know what this type "unknown" is/do.
Unless it has some functionaility I'm not aware of i would rather use "any"